releasenotes/notes/6110-709f6fa891bdb26b.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    For the class :class:`~DagDepNode`, thew previously deprecated ``condition`` attribute have been removed. It was deprecated in 0.18 release.
```

```suggestion
    For the class :class:`~DagDepNode`, the previously deprecated ``condition`` attribute has been removed. It was deprecated in 0.18 release.
```
We should also add a sentence explaining what to do instead.